I'm not sure where to add the tcl and tk libs, so I simply
extend every possible point in the makefile where I find libpython
with $(NAN_PYTHON_EXTRA), which is set in the .conf file.
	--jsaw

--- blender-2.34/source/Makefile~	2004-07-19 04:43:07.000000000 +0200
+++ blender-2.34/source/Makefile	2004-10-27 15:54:24.871424808 +0200
@@ -90,7 +90,7 @@
     GRPLIB += $(NAN_OPENNL)/lib/$(DEBUG_DIR)libopennl.a
     GRPLIB += $(NAN_SUPERLU)/lib/$(DEBUG_DIR)libsuperlu.a
 
-    GRPLIB += $(OCGDIR)/blender/python/$(DEBUG_DIR)libpython.a
+    GRPLIB += $(OCGDIR)/blender/python/$(DEBUG_DIR)libpython.a $(NAN_PYTHON_EXTRA)
 
 # nlin: the reason that some libraries appear more than once below is
 # to handle circular dependencies in linking among libraries... some
@@ -282,7 +282,7 @@
 ifeq ($(OS),darwin)
     BINTARGETS = blenderdynamic.app
     BINTARGETS += blenderplayer.app
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a  $(NAN_PYTHON_EXTRA)
     # Next line would enable MxTextTools (for VRML2 import) 
     # but results in a link error
     # PYLIB += $(NAN_MXTEXTTOOLS) 
@@ -297,7 +297,7 @@
     BINTARGETS += blenderplayer
     BINTARGETS += plugin
   endif
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a  $(NAN_PYTHON_EXTRA)
     PYLIB += $(NAN_MXTEXTTOOLS) 
 endif
 
@@ -305,7 +305,7 @@
     BINTARGETS = blenderdynamic
     BINTARGETS += blenderplayer
     BINTARGETS += plugin
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a  $(NAN_PYTHON_EXTRA)
     PYLIB += $(NAN_MXTEXTTOOLS)
 endif
 
@@ -327,13 +327,13 @@
     BINTARGETS = blenderdynamic
     BINTARGETS += blenderstatic
   endif
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a  $(NAN_PYTHON_EXTRA)
     PYLIB += $(NAN_MXTEXTTOOLS)
 endif
 
 ifeq ($(OS),openbsd)
     BINTARGETS = blenderstatic
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a $(NAN_PYTHON_EXTRA)
 endif
 
 ifeq ($(OS),solaris)
@@ -342,7 +342,7 @@
   ifeq ($(CPU),sparc)
     BINTARGETS += plugin
   endif
-    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
+    PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a $(NAN_PYTHON_EXTRA)
     PYLIB += $(NAN_MXTEXTTOOLS)
     PYLIB += $(NAN_ZLIB)/lib/libz.a
 
